src/SDKs/_metadata/azsadmin_resource-manager_compute.txt

```diff
@@ -1,11 +1,2 @@
-2018-04-11 18:34:50 UTC
+﻿
```

Comment:
Please generate the code using generate.ps1 or generate.cmd and commit those changes.
Looks like autorest failed to generate the code from config file here

Reply:
Please checkout this file and run the code generation script again. These changes are not expected.